app/models/iso_image.rb

```diff
@@ -1,5 +1,5 @@
 class IsoImage < ApplicationRecord
-  belongs_to :iso_datastore
+  belongs_to :storage
```

Comment:
Love this because it opens us up to creating these from other provider storage types like VMware when we do a datastore ssa.  This would make it a lot simpler to get a list of ISOs when doing vm reconfigure to connect a cdrom or provision from an ISO